### PR TITLE
Allow commands not starting with `db.<commandName>` to also be parsed and executed from shell

### LINKED ISSUE #565: Cannot run queries on secondary.
Repro steps:
1. Connect to a secondary on a standard MongoDB replicated cluster
2. try to query a document

I tried running rs.slaveOk() from a mongo scrapbook, but that didn't work. am I missing something?

Action: cosmosDB.executeMongoCommand
Error type: 13435
Error Message: not master and slaveOk=false

Version: 0.6.0
OS: linux

### PR DESCRIPTION
**Pull request by PrashanthCorp**
Fix #565

Of the 6 files, the only file manually generated is the mongo.g4 file. That has the spec for the language we intend to parse. Other files are auto-generated on `npm run update-grammar`, `npm i`, etc. 

The bug was about allowing `rs.slaveOk` - the docs detail many other commands starting with `sh.`  and `BULK.` I wanted to generalize it a bit, without having to change a lot of the code in MongoScrapbook.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -699,9 +699,9 @@
 			"integrity": "sha1-rFcX8w8++jYXsATo/0+GC5x9TSA="
 		},
 		"antlr4ts-cli": {
-			"version": "0.4.0-canary.37504efa",
```
Comment (EricJizbaMSFT) [inline package-lock.json]: How come this changed?

Comment (PrashanthCorp) [reply package-lock.json]: I ran an npm i to remove any changes to my package.json from the management place API  PR .

File: grammar/mongo.g4 [hunk 2]
```diff
@@ -19,7 +19,9 @@ mongoCommands: commands EOF;
 
 commands: ( command | emptyCommand | comment)*;
 
-command: DB (DOT collection)? (DOT functionCall)+ SEMICOLON?;
+command: ((DB (DOT collection)?) | 'rs' | 'sh' | 'BULK') (
```
Comment (StephenWeatherford) [inline grammar/mongo.g4]: Can you refactor this to make it easier to understand?

File: grammar/mongo.g4 [hunk 3]
```diff
@@ -19,7 +19,9 @@ mongoCommands: commands EOF;
...
+		DOT functionCall
+	)+ SEMICOLON?;
 
 emptyCommand: SEMICOLON;
```
Comment (StephenWeatherford) [inline grammar/mongo.g4]: Unit and/or CTI tests?

**Comment by EricJizbaMSFT** [comment]
Closing due to lack of activity. Feel free to re-open once you're actively working on this

**Pull request closed without merging**